docs/tutorials/tutorials-runtime-upgrade.md

```diff
@@ -38,6 +38,8 @@ sidebar_label: Runtime Upgrade
 
 ![Submit Preimage](/img/tutorial/submit_preimage.png "Submit Preimage")
 
+**Important** If you forgot to copy the preimage hash, you can easily regenerate and copy it again by choosing the same extrinsic.(It won't allow to submit it again but you can get preimage hash.).
```

Comment:
could we please change `extrinsic.(It` to `extrinsic (it`, 
and change `hash.).` to `hash).`